inlong-manager/manager-schedule/src/test/java/org/apache/inlong/manager/schedule/airflow/AirflowContainerEnv.java

```diff
@@ -0,0 +1,124 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.inlong.manager.schedule.airflow;
+
+import lombok.extern.slf4j.Slf4j;
+import okhttp3.OkHttpClient;
+import okhttp3.Request;
+import okhttp3.Response;
+import org.testcontainers.containers.ContainerState;
+import org.testcontainers.containers.DockerComposeContainer;
+import org.testcontainers.utility.MountableFile;
+
+import java.io.File;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Optional;
+
+@Slf4j
+public class AirflowContainerEnv {
+
+    public static String BASE_URL = "http://localhost:8080";
+    public static String AIRFLOW_USERNAME = "airflow";
+    public static String AIRFLOW_PASSWORD = "airflow";
+    public static String NORMAL_POSTFIX = "_normal";
+    public static String CORN_POSTFIX = "_cron";
+    public static String DOCKER_COMPOSE_YAML_PATH = "src/test/resources/airflow/docker-compose.yaml";
+
+    private static DockerComposeContainer<?> environment;
+    private static OkHttpClient httpClient = new OkHttpClient();
+
+    public static void setUp() throws InterruptedException {
```

Comment:
The current method shows that the possible exception thrown may be `InterruptedException`, but the `waitForDAGLoad` method throws a `RuntimeException` instead

Reply:
done.